CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+## Development
+
+This doc explains how to set up a development environment so you can get started contributing to this project
+
+## Prerequisites
+
+1. [`go`](https://golang.org/doc/install): For building the project
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`gcloud`](https://cloud.google.com/sdk/docs/install): For Interacting with the Google Cloud Platform
+
+This guides breaks down into two parts 
+1. Setting up the GKE cluster - If you have already setup the GKE cluster, you can skip this part
+2. Setting up the development environment - This part is required for all the contributors
+
+## Setting up the GKE cluster
+
+1. First Go ahead and create new project in the Google Cloud Platform and enable billing for the project if you don't have it already enabled. You can follow the below link to create a new project
+
+
+2. Once you have created the project enable the below apis 
+
+```
+gcloud services enable compute.googleapis.com
```

Comment:
align

Reply:
@jwcesign  I don't understand what you meant here , can you tell me how you want me to align

Reply:
2. Once you have created the project enable the below apis 
    ```
    gcloud services enable compute.googleapis.com
    ```
 
 Not:
 2. Once you have created the project enable the below apis 

```
gcloud services enable compute.googleapis.com
```